### ISSUE TITLE
FR: RMB+Drag Panning

### ISSUE THREAD

**Issue by kosirm** [zsviczian/obsidian-excalidraw-plugin#1444]
**Is your feature request related to a problem? Please describe.**
Switching between 1,2,3,... (Tools) and H (Panning) is counterintuitive and it is used 90% of the time.

**Describe the solution you'd like**
RMB+Drag is the most intuitive solution IMHO. The context menu should be moved from the RMB mouseup event to the RMB click event and then RMB drag can be used for Panning. **After panning focus should be back on the current tool.**

**Describe alternatives you've considered**
An alternative is MMB drag, which is not working well for me (maybe my mouse is a problem). Another alternative is holding some keyboard button (for example space) + LMB drag, but **after panning focus should be back on the current tool**. In most drawing programs panning is a transient event.

**Additional context**
Just a huge gratitude for your hard work with this genius plugin.

**Comment by kosirm** [comment]
Definitely, my mouse MMB is a problem, it's very hard to press. So maybe it's just me.

**Comment by zsviczian** [comment]
SPACE + LMB works
another alternative is "h" for the hand tool, then LMB to drag

I recommend opening this issue on the core Excalidraw product site, this feature should be addressed in the package, not the plugin. https://github.com/excalidraw/excalidraw/issues

**Comment by kosirm** [comment]
Just what I was looking for! Perfect! Thanks, @zsviczian
